hawc/apps/form_library/forms.py

```diff
@@ -0,0 +1,41 @@
+from django import forms
+from django.urls import reverse
+
+from hawc.apps.common.autocomplete.forms import AutocompleteSelectMultipleWidget
+from hawc.apps.common.dynamic_forms.schemas import Schema
+from hawc.apps.common.forms import BaseFormHelper, PydanticValidator
+from hawc.apps.myuser.autocomplete import UserAutocomplete
+
+from .models import CustomDataExtraction
+
+
+class CustomDataExtractionForm(forms.ModelForm):
+    schema = forms.JSONField(
+        initial=Schema(fields=[]).dict(),
+        validators=[PydanticValidator(Schema)],
+    )
+
+    def __init__(self, *args, **kwargs):
+        self.user = kwargs.pop("user")
+        super().__init__(*args, **kwargs)
+        self.fields["creator"].initial = self.user
+
+    class Meta:
+        model = CustomDataExtraction
+        exclude = ("parent_form", "created", "last_updated")
+        widgets = {
+            "editors": AutocompleteSelectMultipleWidget(UserAutocomplete),
+            "creator": forms.HiddenInput,
```

Comment:
Even though it's hidden, it can be modified. I changed so it's set server-side based on the request.

Reply:
I think you can remove the widget? and exclude the field?

Reply:
resolved; this was done